### DISCUSSION TITLE
I'm a developer

### DISCUSSION THREAD

**Discussion by DanStuartDept** [six7/figma-tokens#1329] [Q&A] [answered]
Hi,

My designer colleague is experimenting with this Figma Tokens plugin and has provided me with a JSON export. 

I'd like to create a script that parses this JSON and converts the tokens to tailwind config. But does anybody know are there tools out there that already do this? Is there a robust example I can follow?  

Thanks,

Dan

**Comment by six7** [comment, marked as answer]
https://github.com/six7/figma-tokens-example-tailwindcss

This one does what you described 👍 

You might need to extend it to support other token types than just color but the principle is the same

**Reply by DanStuartDept** [reply]
Thanks @six7 I will have a look into this
